packages/react/src/form/fieldset/fieldset.stories.tsx

```diff
@@ -51,13 +51,13 @@ export const FieldsetError: Story = {
   },
   render: (props) => (
     <Fieldset {...props}>
-      <Checkbox hasError value="Hello">
+      <Checkbox errorMessage value="Hello">
         Hello
       </Checkbox>
-      <Checkbox hasError value="Hello">
+      <Checkbox errorMessage value="Hello">
         Hello
       </Checkbox>
-      <Checkbox hasError value="Hello">
+      <Checkbox errorMessage value="Hello">
         Hello
       </Checkbox>
     </Fieldset>
```

Comment:
🔧 Could another api using an optional `FieldsetContext` that that Checkbox and Radiobuttons tries to use, and it can read `errorMessage` from there?

Reply:
Could even do it on the css level, if not to much hassle.

Adding `hds-fieldset--error` to the selectors.
```css
  .hds-fieldset--error,
  .hds-checkbox.hds-checkbox--error {
    .hds-checkbox__checkmark,
    .hds-checkbox__checkmark::after {
      border-color: var(--hds-ui-colors-warning-yellow);
    }
  }
```

Reply:
Using a context would also remove the need for `isStandalone` as prop 🤷

Reply:
As we discussed in the office today, I have now changed the Checkbox to accept _either_ `hasError` or a new prop `errorMessage`.

I will continue to work on the context, but that will naturally build on what I have done so far in this PR, so I suggest we could merge this now, and introduce the fieldset context very soon.

I have updated the PR description and the changeset accordingly.